firmware/library/L1_Drivers/button_driver.hpp

```diff
@@ -1,10 +1,10 @@
-#pragma once
-
-class ButtonInterface {
- public:
-         virtual void Init()                = 0;
-         virtual bool Read()                = 0;
-         virtual bool Released()            = 0;
-         virtual bool Pressed()             = 0;
-         virtual void InvertButtonSignal(bool enable_invert_signal = true) = 0;
-};
+#pragma once
+
+class ButtonInterface {
+ public:
+         virtual void Init()                = 0;
+         virtual bool Read()                = 0;
+         virtual bool Released()            = 0;
+         virtual bool Pressed()             = 0;
+         virtual void InvertButtonSignal(bool enable_invert_signal = true) = 0;

```

Comment:
If you have some means of changing how Button signals are interpreted, it would necessary to also have a getter for this. Otherwise, the developer will be guessing at whether or not a button had been inverted or not.

Reply:
I don't know how this commented ended up here.

Reply:
I'm not working on this driver, but this ended up in my differences for some reason, I'll see if I can get this cleaned up right now.